src/components/loading/loading.js

```diff
@@ -1,16 +1,13 @@
-import { css } from '@emotion/react';
 import PuffLoader from 'react-spinners/PuffLoader';
+import "./loading.css"
 
-// Can be a string as well. Need to ensure each key-value pair ends with ;
-const override = css`
-  display: block;
-  margin: 0 auto;
-`;
+const color = '#7A7A7A'
 
 const Loading = () => {
   return (
-    <div>
-      <PuffLoader css={override} size={60} />
+    <div className='container-loading'>
+      <PuffLoader size={160} color={color} />
```

Comment:
Como no estan usando esta variable `color` en otros lugares. Creo que no es necesario crear una variable.

```suggestion
      <PuffLoader size={160} color="#7A7A7A" />
```

¿Qué opinan ustedes?

Reply:
***c6231c109871a2ca93e66bfa807f4bb0e7f23644*** listo!